backend/config/corpora-api.yml

```diff
@@ -604,6 +604,54 @@ paths:
         "403":
           $ref: "#/components/responses/403"
 
+  /dp/v1/datasets/index:
+    get:
+      tags:
+        - datasets
+      summary: List datasets for indexing purposes
```

Comment:
```suggestion
      summary: List non-tombstoned datasets for indexing purposes
```

Reply:
`for indexing purposes` isn't entirely clear to me; maybe `for client-side user filtering`?  (though DP API is used by some collaborators, I think it's okay to continue to think of the current API as a backend specifically for the UI; a public API will probably be different...more RESTful, etc.).

Reply:
Yeah, I will rewrite this. I think the key point to convey is that this endpoint isn't meant for a general usage because it lacks some fields that aren't required by the index we're trying to create, nor by displaying the dataset in the compact view.